Readme.md

```diff
@@ -15,7 +15,7 @@ Please read the [FAQ](https://dolphin-emu.org/docs/faq/) before using Dolphin.
 * OS
     * Windows (10 or higher).
     * Linux.
-    * macOS (10.14 Mojave or higher).
+    * macOS (10.15 or higher).
```

Comment:
I'd probably still add Catalina here as it's technically the name of the OS.

Reply:
I think we should include it. It's not just an internal codename, it's user-facing.